### DISCUSSION TITLE
licença do Chrome para baixar vídeos do YouTube

### DISCUSSION THREAD

**Discussion by devilsandro** [aclap-dev/video-downloadhelper#1278] [License & Purchase requests]
### Your question

Good evening, I bought a Chrome license yesterday to download videos from YouTube, but to my surprise it is not possible, it has the following message: (Unfortunately, the Chrome browser store does not allow video transfers, so we cannot include this function in the version intended for Chrome.) Is it possible to switch to the edge browser ?or extort the value? I already have the extension in the Firefox browser but I need it in another browser or if it works in Chrome, could someone please contact me, the email is: [alexandreards@hotmail.com](mailto:alexandreards@hotmail.com)
![unnamed](https://github.com/user-attachments/assets/34883e04-78b6-4204-a41d-eb7a77a77e54)

**Comment by jcfrog** [comment]
Hi,
I sent you an email.

**Comment by devilsandro** [comment]
This is proof of how I can get the refund as it was done via credit card

________________________________
De: jcfrog ***@***.***>
Enviado: quarta-feira, 18 de setembro de 2024 04:54
Para: aclap-dev/video-downloadhelper ***@***.***>
Cc: devilsandro ***@***.***>; Author ***@***.***>
Assunto: Re: [aclap-dev/video-downloadhelper] licença do Chrome para baixar vídeos do YouTube (Discussion #1278)


Closed #1278<https://github.com/aclap-dev/video-downloadhelper/discussions/1278> as resolved.
